server/src/main/java/org/apache/iotdb/db/query/executor/AggregationExecutor.java

```diff
@@ -121,23 +127,21 @@ private AggregateResult aggregateOneSeries(int i, Filter timeFilter, QueryContex
         //cal by pageheader
         if (seriesReader.canUseCurrentPageStatistics()) {
           Statistics pageStatistic = seriesReader.currentPageStatistics();
-          aggregateResult.updateResultFromStatistics(pageStatistic);
-          if (aggregateResult.isCalculatedAggregationResult()) {
```

Comment:
If one series relates to a   min_time and first_value, we'd better return result if all aggregateResult  has calculated aggregation results.

Reply:
Thanks for your proposal. I use an `aggregateResultList` to store whether the `aggregateResult` is calculated... which is consistent with the `aggregateResultList`. Do you think it is okay?